egs/commonvoice/ASR/local/compute_fbank_commonvoice_splits.py

```diff
@@ -41,6 +43,13 @@
 def get_args():
     parser = argparse.ArgumentParser()
 
+    parser.add_argument(
+        "--subset",
+        type=str,
+        default="train",
```

Comment:
Could you add a `choices` here? Otherwise, it is not clear what values are valid.